rules/require-sections.js

```diff
@@ -0,0 +1,68 @@
+var rule = require('unified-lint-rule')
+var position = require('unist-util-position')
+var schema = require('../util/schema')
+
+module.exports = rule('standard-readme:require-sections', requireSections)
+
+/* Max allowed file-size without a table of contents. */
+var maxLinesWithoutTableOfContents = 100
+
+/* Pretty heading names people *should* use. */
```

Comment:
Add an error for using 'installation', 'How to use', 'Contributing', and 'licensing'. These are common errors which should be caught and hinted.

Reply:
I think that should be another rule!

Because the “extra” placeholder can contain any heading though, people are free to use all those headings though, and by doing so “enter” the “extra” placeholder 🤷‍♂️

Reply:
Fair enough!